languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -13645,6 +13645,24 @@ though, and has since been slightly modified)-->
         </rulegroup>
     </category>
     <category id="CAT_GRAMMAIRE" name="Grammaire">
+        <rule id="MULTI_ADJ" name="accord entre plusieurs adjectifs">
+            <pattern>
+                <unify negate="yes">
+                    <feature id="number"/>
+                    <feature id="gender"/>
+                    <token postag="J.*" postag_regexp="yes"/>
+                <unify-ignore>
+                    <token min="0" max="1">,</token>
+                    <token>et|ni</token>
+                </unify-ignore>
+                    <token postag="J.*" postag_regexp="yes"/>
+                </unify>
+            </pattern>
+            <message>L'adjectif "\4" ne semble pas accordé</message>
+            <suggestion><match no="4" postag="J*" postag_regexp="yes" postag_replace="$1"></match></suggestion>
```

Comment:
`J*` → `J.*`

The `$1` dans `postag_replace` doesn't know what to refer to

<match></match> is an "empty string" → <match/>